apps/namadillo/src/atoms/proposals/functions.ts

```diff
@@ -257,16 +258,16 @@ const fromIndexerStatus = (
 
 const toIndexerStatus = (
   proposalStatus: ProposalStatus
-): IndexerProposalStatusEnum => {
+): ApiIndexerProposalStatusEnum => {
```

Comment:
So there are two enums specified in swagger file. One is for status used in query params and other is used as a part of proposal response type. I think they should always be the same(kind of), but in rust we have two, because query params and response are two separate things.
FIrst:
```
        status:
          type: string
          enum: [pending, voting, passed, rejected]
```
Second:
```
          schema:
            type: string
            enum: [pending, votingPeriod, passed, rejected]
```
It's a bit confusing :D We can potentially leave two enums in rust but have only one described in swagger 🤔